build.gradle

```diff
@@ -64,6 +64,28 @@ dependencies {
     testImplementation "junit:junit:$junit_version"
 }
 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+    implementation "io.swagger:swagger-parser:$swagger_parser_version"
+    implementation "io.swagger:swagger-core:$swagger_core_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    testImplementation "junit:junit:$junit_version"
+}
 buildscript {
```

Comment:
Are both buildscript sections needed here?

Reply:
I don't think so, I just missed that.